.obs/dockerfile/SL-Micro-base-container/Dockerfile.iso

```diff
@@ -10,7 +10,7 @@
 ARG SLMICRO_VERSION
 
 FROM suse/sl-micro/${SLMICRO_VERSION}/base-os-container:latest AS os
-FROM suse/sl-micro/${SLMICRO_VERSION}/baremetal-os-container:latest AS builder
+FROM suse/sl-micro/${SLMICRO_VERSION}/base-os-container:latest AS builder
```

Comment:
Using the OS image as the builder itself simplifies the image cross dependencies, so the ISO only requires the OS container to build, so `base` ISO only requires `base` OS.